File: pascal/fizzbuzz.pas
program fizzbuzz;

var i:integer;

begin
  for i := 0 to 100 do
    if (i mod 5 = 0) and (i mod 3 = 0) then
      writeln('Fizzbuzz')
    else if i mod 5 = 0 then
      writeln('Buzz')
    else if i mod 3 = 0 then
      writeln('Fizz')
    else
      writeln(i);
end.

File: ruby/fizzbuzz.rb
#!/usr/bin/env ruby

for i in 1..100
  if i % 15 == 0
    print "FizzBuzz, "
  elsif i % 5 == 0
    print "Buzz, "
  elsif i % 3 == 0
    print "Fizz, "
  else
    print i.to_s << ", "
  end
end
